packages/amplify_core/analysis_options.yaml

```diff
@@ -5,4 +5,3 @@ analyzer:
     public_member_api_docs: ignore
   exclude:
     - lib/**/*.g.dart
-    - doc/
```

Comment:
Can you verify we don't lose any pub points from analysis by changing this? Looks like we're currently [130/140](https://pub.dev/packages/amplify_core/score), but lets avoid a regression if possible.

Reply:
The goal is ultimately to not require any excludes which will help us catch analysis errors in CI which may show up in pub. Removing this exclude will help make sure our pub points remain high 👍